src/Common/src/TypeSystem/Common/TargetDetails.cs

```diff
@@ -174,6 +174,19 @@ public int MinimumFunctionAlignment
             }
         }
 
+        /// <summary>
+        /// Gets the alignment that is optimal for this platform.
+        /// </summary>
+        public int OptimalFunctionAlignment
+        {
+            get
+            {
+                // Matches the choice in the Project N compiler.
```

Comment:
I would rather snap to defaults for C/C++ compilers. Is this the default for x86 in C/C++ compilers?

Reply:
Yes, 16 is the default for x86 in C++ as well.